operations/kubernetes/Retrieve_Cluster_Health_Information_Using_Kubernetes.md

```diff
@@ -21,134 +22,81 @@ ncn-w002   Ready    <none>                 8d    v1.20.13
 ncn-w003   Ready    <none>                 8d    v1.20.13
 ```
 
-### Retrieve Pod Status
+## Pods
+
+### Retrieve information about individual pods
 
 ```bash
-ncn# kubectl get pods options
+ncn# kubectl describe pod POD_NAME -n NAMESPACE_NAME
 ```
 
-Example output:
+### Retrieve a list of all pods
 
-```
-NAME                                    READY     STATUS    RESTARTS   AGE
-api-gateway-6fffd4d854-btp4v            1/1       Running   2          11d
-api-gateway-6fffd4d854-l7z8m            1/1       Running   1          11d
-api-gateway-6fffd4d854-tqx8v            1/1       Running   1          11d
-api-gateway-database-8555685975-xw5gm   1/1       Running   1          11d
-cray-ars-6466bcf77d-s99rq               1/1       Running   1          11d
-cray-bss-7589d7459f-ttwf4               1/1       Running   1          11d
-cray-capmc-6bf5855b78-np72d             1/1       Running   1          11d
-cray-datastore-768576677d-gpnl7         1/1       Running   1          11d
-cray-dhcp-5fccf85696-wrqxb              1/1       Running   1          11d
-cray-tftp-885cc65c4-568jz               2/2       Running   2          11d
-kvstore-1-56fdb6574c-ts79v              1/1       Running   1          11d
-kvstore-2-694bc7567b-k99tl              1/1       Running   1          11d
-kvstore-3-5b658b9bd9-n9dgt              1/1       Running   1          11d
+```bash
+ncn# kubectl get pods -A
 ```
 
-### Retrieve Information about Individual Pods
+### Retrieve a list of healthy pods
 
 ```bash
-ncn# kubectl describe pod POD_NAME -n NAMESPACE_NAME
+ncn# kubectl get pods -A | grep -E 'Completed|Running'
 ```
 
-### Retrieve a List of Healthy Pods
+### Retrieve a list of unhealthy pods
 
 ```bash
-ncn# kubectl get pods -A | grep -e 'Completed|Running'
+ncn# kubectl get pods -A | grep -Ev 'Completed|Running'
 ```
 
-Example output:
+or
```

Comment:
I am confused by this "or"

Reply:
It is showing two different ways to look for unhealthy pods. I will add some language to clarify this.